src/frontend/src/binder/struct_field.rs

```diff
@@ -50,7 +50,7 @@ impl Binder {
                 };
                 let index = self
                     .context
-                    .get_column_binding_index(&Some(table_name), &column)?;
+                    .get_column_binding_index(&Some(table_name), &column)?[0];
```

Comment:
Is this always the right thing to do? Perhaps we should throw an error here if we encounter multiple indices (ambiguous reference).

But I think its fine as it is impossible for there to be ambiguous reference in struct?